Micro16C.Web/src/Micro16C.Web.Client/Main.fs

```diff
@@ -0,0 +1,163 @@
+module Micro16C.Web.Client.Main
+
+open System
+open Elmish
+open Bolero
+open Bolero.Html
+open System.IO
+open Micro16C.Backend
+open Micro16C.Frontend
+open Micro16C.MiddleEnd
+open Micro16C.MiddleEnd.PassManager
+
+let private debugModulePasses title =
```

Comment:
I realize you copied this from Program.fs in Micro16C. Would it maybe be possible to refactor `compile` into a solution that both Micro16C.Web.Client and Micro16C depend on?

Reply:
Done, hopefully I didn't screw up this part too much